### DISCUSSION TITLE
Bind tempel-next to M-L does not work

### DISCUSSION THREAD

**Discussion by thorstengrothe** [minad/tempel#165] [Q&A] [answered]
Hi,

first of all, thank you very much for this great package. I'm planing to use it completely for all my snippets, mostly for LaTeX and remove yasnippet from my config.

I would like to bind `tempel-next` to `M-l` and `tempel-previous` to `M-J` like so

```
(use-package tempel
  ;; Require trigger prefix before template name when completing.
  :custom
  (tempel-trigger-prefix "j")
  :bind (("M-y" . tempel-complete)
         :map tempel-map
         ("M-l" . tempel-next)
         ;; ("<tab>" . tempel-next) --> this works
         ("M-j" . tempel-previous)
        ("<backspace>" . tempel-previous) --> this works
         ...
```
But it does not work for me. <tab> and <backspace> are working fine BTW. What I'm doing wrong?

Regards Thorsten

**Comment by minad** [comment]
I can only guess. Maybe some other key binding shadows M-l or M-j? Use `C-h k M-l` to check when Tempel is active.

**Comment by thorstengrothe** [comment, marked as answer]
I got it working now. It was an error with the tempel package. I guess it was too old. I upgraded with elpaca and now it works fine. Thanks for your help and sorry for the noise!
